test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/probe/ReadinessProbeIntegrationTest.java

```diff
@@ -116,7 +131,7 @@ public void testKieServerReadinessProbe() {
         while (Calendar.getInstance().getTimeInMillis() < timeoutTime) {
             Response response = target.request().get();
 
-            if(response.getStatus() == Response.Status.OK.getStatusCode()) {
+            if (response.getStatus() == Response.Status.OK.getStatusCode()) {
```

Comment:
Having if in the test seems evil. How about assert? :)
I realized you didn't implement this in this PR, but for the future I'd break it up by first waiting for OK response and then continuing with those asserts (to getting rid of the branching)